source/sass/fonts.scss

```
@@ -1,22 +1,25 @@
 @font-face {
   font-family: "Open Sans";
-  src: url("../fonts/opensans.woff2") format("woff2");
+  src: url("../fonts/opensans.woff2") format("woff2"),
+    url("../fonts/opensans.woff") format("woff");
```

Я обычно так делаю, чтобы и линтер не ругался, и строки вместе стартовали:

```suggestion
  src:
    url("../fonts/opensans.woff2") format("woff2"),
    url("../fonts/opensans.woff") format("woff");
```

А ты молодец, все отлично со шрифтами